Спринт 6, Юнит тесты

eusatenko opened the pull request:
Готово. 
Разделил параметризованные тесты и обычные по разным классам.

eusatenko committed:
Спринт 6, Юнит тесты

egorvas commented on .idea/.gitignore:
```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```
Рекомендую добавлять папку .idea в .gitignore всего репозитория. По сути файлы в папке .idea это твои локальный файлы и не нужны для работоспособности проекта, но при этом могут приватные данные.

eusatenko committed:
1) Переделал параметры и assert в AnimalParameterizedTest.java
2) Добавил в .gitignore папки .idea и target
3) Добавил target/site/ как исключение в .gitignore
